homework6/task1.py

```diff
@@ -0,0 +1,52 @@
+# Напишите программу вычисления арифметического выражения заданного строкой. 
+# Используйте операции +,-,/,*. приоритет операций стандартный.
+
+#  Пример:
+#  2+2 => 4;
+#  1+2*3 => 7;
+#  1-2*3 => -5;
+
+# Добавьте возможность использования скобок, меняющих приоритет операций.
+#  Пример:
+#  1+2*3 => 7;
+#  (1+2)*3 => 9;
+
+
+
+expression = input('Введите арифметическое выражение: ')
+
+def parse(expression):
+    num = ''
+    exp = []
+    for char in expression:
+        if char.isdigit() or char == '.':
+            num += char
+        elif char == ' ':
+            continue
+        else:
+            exp.append(float(num))
+            num = ''
+        if char in '/+-*':
+                exp.append(char)
+    if num:
+            exp.append(float(num))
+    return exp
+
+def calculator(exp):
+    for i in exp:
```

Comment:
Для вашего кода должно быть так
```suggestion
    for i in range(len(exp)):
```
Это решит проблему ошибки, но не проблему неверного вычисления.